src/main/resources/templates/topic-detail.ftlh

```diff
@@ -156,7 +156,7 @@
                 <tbody>
                 <#list consumers![] as c>
                     <tr>
-                        <td><a href="<@spring.url '/consumer/${c.groupId}'/>">${c.groupId}</a></td>
+                        <td><a href="<@spring.url '/consumer/${c.groupIdBase64}'/>">${c.groupId}</a></td>
```

Comment:
Instead of using a base64 encoding, why not just url encode it? In this way it should be handled automatically by spring I suppose. Probably you can use `UriComponentsBuilder`. In this way you don't need to change `src/main/java/kafdrop/controller/ConsumerController.java`.

Reply:
I tried that way as well, however `spring.url` was again converting it to decoded value, which was ending up with error when you click on consumer group link.